### ISSUE TITLE
Error when handling **kwargs parameters

### ISSUE THREAD

**Issue by adrienhenry** [pawamoy/pytkdocs#20]
**Describe the bug**
I get the following error when I document `**kwargs` parameter in my docstring with the two `**`.
```
line 216, in read_parameters_section
    signature_param = self.signature.parameters[name]
KeyError: '**kwargs'
```
**To Reproduce**
```python
def function(a, **kwargs):
    """
    Args:
        a: a parameter.
        **kwargs: kwarg parameters
    """
    return None
```

**Information (please complete the following information):**
- OS: Debian
- `mkdocstrings` version: [0.7.0] 

**Additional context**
This seems to come from the *inspect* module as `inspect.signature(function).parameters` would return:
```
mappingproxy({'a': <Parameter "a">, 'kwargs': <Parameter "**kwargs">})
```
and the `**kwargs` key is not present but `kwargs` is.

**Comment by adrienhenry** [comment]
Can I help with something?
The `signature_param = self.signature.parameters[name]` to `signature_param = self.signature.parameters[name.lstrip("*")]` fix the bug.

**Comment by pawamoy** [comment]
Of course, you can open a PR with this fix if you want!

**Comment by pawamoy** [comment]
Fixed in #21
